### PR TITLE
Feature sign up and login tests

### PR DESCRIPTION
**Pull request by kaymmatthew**

### CODE REVIEW

File: PageObjects/UserSignUpPage.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 {
     public class UserSignUpPage : Hooks
     {
+        private readonly ScenarioContext? _scenarioContext;
```
Comment (GodsTime2023) [inline PageObjects/UserSignUpPage.cs:5]: Can you remove scenario context if not currently been used please.

Comment (GodsTime2023) [inline PageObjects/UserSignUpPage.cs:5]: Remove scenario context

File: POMJulyBatch_Task.csproj [hunk 2]
```diff
@@ -39,6 +39,7 @@
     </PackageReference>
     <PackageReference Include="Selenium.Support" Version="4.11.0" />
     <PackageReference Include="Selenium.WebDriver" Version="4.11.0" />
+    <PackageReference Include="SpecFlow" Version="3.9.74" />
```
Comment (GodsTime2023) [inline POMJulyBatch_Task.csproj:42]: Uninstall specflow package

File: TestModules/KaySignUpTest.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+            userSignUpPage.FillUserAccountInformation(jsonData.GetData("UserInfo:password"), jsonData.GetData("UserInfo:days"),
+                jsonData.GetData("UserInfo:month"), jsonData.GetData("UserInfo:year"), jsonData.GetData("UserInfo:firstName"),
+                jsonData.GetData("UserInfo:lastName"), jsonData.GetData("UserInfo:company"), jsonData.GetData("UserInfo:address"),
+                jsonData.GetData("UserInfo:address2"), jsonData.GetData("UserInfo:country"), jsonData.GetData("UserInfo:state"),
+                jsonData.GetData("UserInfo:city"), jsonData.GetData("UserInfo:zipCode"), jsonData.GetData("UserInfo:mobile"));
+
+            homePg.ClickSigUpLkn();
+
+            Assert.That(userName, Is.Not.Empty);
```
Comment (GodsTime2023) [inline TestModules/KaySignUpTest.cs:37]: Were is the element used for assertion

File: PageObjects/UserSignUpPage.cs (hunk 1 above)
Comment (kaymmatthew) [reply PageObjects/UserSignUpPage.cs]: Resolved

File: POMJulyBatch_Task.csproj (hunk 2 above)
Comment (kaymmatthew) [reply POMJulyBatch_Task.csproj]: Resolved

**Review by GodsTime2023 — APPROVED** [review]

**Pull request merged**